salem/gis.py

```diff
@@ -938,7 +946,8 @@ def region_of_interest(self, shape=None, geometry=None, grid=None,
                                         inplace=inplace)
             import rasterio
             with rasterio.Env():
-                mask = rasterio.features.rasterize(shape.geometry, out=mask)
+                mask = rasterio.features.rasterize(shape.geometry,
+                                                   **rasterize_kws)
         if geometry is not None:
```

Comment:
Could you also add the kwarg to the rasterize call in the ``if geometry is not None`` block just below?